policyengine_us/parameters/gov/states/az/tax/income/rebates/age_threshold.yaml

```diff
@@ -0,0 +1,10 @@
+description: Arizona provides different amounts of one-time families tax rebates based on this age_threshold. 
+values:
```

Comment:
Don't need this file, we can just use the age parameter threshold from the scale